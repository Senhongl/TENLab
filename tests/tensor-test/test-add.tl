a = [1,2,4];
b = [2,3,4];
print(a+b);
print(a+-b);
print(a+1);
