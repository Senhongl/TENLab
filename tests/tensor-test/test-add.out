 3
 5
 8
[ CPUIntType{3} ]
-1
-1
 0
[ CPUIntType{3} ]
 2
 3
 5
[ CPUIntType{3} ]